fe/fe-core/src/main/java/com/starrocks/sql/ast/PartitionNames.java

```diff
@@ -49,28 +51,40 @@ public class PartitionNames implements ParseNode, Writable {
     @SerializedName(value = "isTemp")
     private final boolean isTemp;
 
+    private List<Expr> partitionValues = new ArrayList<>();
```

Comment:
Please try not to introduce new name, why not just reuse `PartitionKey`, 
`fe/fe-core/src/main/java/com/starrocks/catalog/PartitionKey.java`